### PR TITLE
wdio-cli: add -y/--yes param to fill in all defaults for wdio config

### PR DESCRIPTION
**Pull request by schuttsm**
## Proposed changes

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)
https://github.com/webdriverio/webdriverio/issues/4522

This adds a -y or --yes flag to wdio-cli config that fills in the default values.  I ran it both with the flag & without (but manually checking all defaults) and the wdio.conf.js files were identical

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

**Comment by schuttsm** [comment]
@webdriverio/technical-committee

todo: add tests around this change.  I wasn't sure if I should add them in wdio-cli or wdio-config packages.  The changes are all in wdio-cli, but its a wdio-config change and I'm not 100% certain where/how I should do it.  Any guidance would be appreciated

File: packages/wdio-cli/src/setup.js [hunk 1]
```diff
@@ -9,10 +9,24 @@ import { CONFIG_HELPER_INTRO, CONFIG_HELPER_SUCCESS_MESSAGE, QUESTIONNAIRE } fro
...
-export default async function setup (exit = true) {
+export default async function setup (params, exit = true) {
     try {
+        var answers
         console.log(CONFIG_HELPER_INTRO)
-        const answers = await inquirer.prompt(QUESTIONNAIRE)
+        if (params.yes) {
+            QUESTIONNAIRE.forEach((question) => {
+                answers = answers || {}
+                if (question.when && !question.when(answers)) return
```
Comment (christian-bromann) [inline packages/wdio-cli/src/setup.js]:
```suggestion
                if (question.when && !question.when(answers)) {
                    return
                }
```

File: packages/wdio-cli/src/setup.js [hunk 2]
```diff
@@ -9,10 +9,24 @@ import { CONFIG_HELPER_INTRO, CONFIG_HELPER_SUCCESS_MESSAGE, QUESTIONNAIRE } fro
...
+
+                if (question.default)
```
Comment (christian-bromann) [inline packages/wdio-cli/src/setup.js]: please use brackets here

File: packages/wdio-cli/src/setup.js [hunk 3]
```diff
@@ -9,10 +9,24 @@ import { CONFIG_HELPER_INTRO, CONFIG_HELPER_SUCCESS_MESSAGE, QUESTIONNAIRE } fro
 import { getNpmPackageName, getPackageName, addServiceDeps } from './utils'
 
 /* istanbul ignore next */
-export default async function setup (exit = true) {
+export default async function setup (params, exit = true) {
     try {
+        var answers
```
Comment (baruchvlz) [inline packages/wdio-cli/src/setup.js]: Use `let` instead.

**Comment by christian-bromann** [comment]
@schuttsm please rebase your branch against master, there we some changes applied to the CLI

**Comment by schuttsm** [comment]
Arg!  Bad merge.  Let me copy the changes over with a fresh branch

**Pull request closed without merging**